releases/8.0.0.md

```diff
@@ -0,0 +1,37 @@
+Arches 8.0.0 release notes
+------------------------
+
+### Major enhancements
+
+
+### Additional highlights
+
+
+### Dependency changes
+```
+Python:
+    Upgraded:
+
+    Added:
+
+    Removed:
+
+
+JavaScript:
+    Upgraded:
+
+    Added:
+
+    Removed:
+```
+
+### Breaking changes
+- The `TileModel` entity has been updated. Instead of `nodegroup` acting as a foreign key to the `Nodegroup` table, it has been replaced with `nodegroup_id`, a simple UUID field. There is now a `nodegroup` class method that will retrieve the Nodegroup from the `nodegroup_id` field, but any direct database references to `nodegroup` should be updated.
+
+### Upgrading Arches
+
+1. You must be upgraded to at least version   before proceeding. If you are on an earlier version, please refer to the upgrade process in the []()
+
+### Upgrading an Arches project
+
+1. Create editable_future_graphs for your Resource Models using the command `python manage.py graph create_editable_future_graphs`. This will publish new versions of each Graph.
```

Comment:
For a graph with a slug, after running this command, I cannot reverse migrate:

```py
django.db.utils.IntegrityError: could not create unique index "graphs_slug_32b94acf_uniq"
DETAIL:  Key (slug)=(my_slug) is duplicated.
```

Reply:
Could you include the steps to repro please?

Reply:
1. Graph Manager > Add a slug and save (field is called "URI Slug for API Access")
2. Create new publication
3. Check db shell: you should have 2 graph rows with the slug you created
4. Reverse migrate: `python manage.py migrate models 9511`

Result: django attempts to reenforce the unique constraint on slug, but PG complains that multiple rows exist

I found this in a more roundabout way just loading AfS on 7.6 and upgrading to 8.0 and running `manage.py graph create_editable_future_graphs` which gave me the duplicate slugs.

Reply:
This was a good catch && took some thinking on how to work around it. Should be fixed in latest